packages/bpk-component-rating/src/BpkRating.scss

```diff
@@ -56,7 +56,12 @@
     justify-content: center;
 
     &--horizontal {
+      // We could use `padding-inline-start: $bpk-spacing-sm` here, but it's unsupported in IE
```

Comment:
Oh I didn't know there was a property for that, that's cool. I really like the 'start' and 'end' ones in RN.